src/components/grid-view/GridViewSlot.scss

```diff
@@ -0,0 +1,15 @@
+@import '../../elements/common/variables';
+@import '../../styles/variables';
+
+.bdl-GridViewSlot {
+    border: 1px solid $efive;
```

Comment:
Should switch to the closest match from here https://opensource.box.com/box-ui-elements/#/Colors (inside the imported variables)